config/appContainer.php

```diff
@@ -28,8 +32,12 @@
 $containerBuilder->register('listener.response', ResponseListener::class)
     ->setArguments(['UTF-8'])
 ;
+$containerBuilder->register('application.error.handler', \Infrastructure\Services\ErrorHandlerHandler::class);
```

Comment:
What should I do if I need extra one handler/listener?

Reply:
Handler is one.  And is reserved service for listener. Infrastructure reserved service to be overriden for custom realization.
Listeners are a lot of.
Than read "read me".
I don' t see any documentation you created in that repo. Maybe you will start? as all that infra is black box and no one knows how to use it.

We have separate project for create project. There i'll add description when you will give me that repo. I asked you. Now I added description in our e-com.